.gitignore

```diff
@@ -1,13 +1,38 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
 node_modules
-.DS_Store
+*.local
+
+# Editor directories and files
+.vscode/*
+!.vscode/extensions.json
 .idea
-.nvmrc
-.docusaurus
+.DS_Store
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw?
```

Comment:
Those entries are totally unrelated to the project.

Reply:
included these from the vite template but sure i'll get rid of em

Reply:
on second thought, those are actually node config files for visual studio so they should be ignored

Reply:
for reference https://github.com/vitejs/vite/blob/main/packages/create-vite/template-react-ts/_gitignore